.travis.yml

```diff
@@ -533,4 +533,5 @@ script:
   - if test -n "$SWIGLANG"; then make $SWIGJOBS check-$SWIGLANG-test-suite CFLAGS="$cflags" CXXFLAGS="$cxxflags"; fi
   - echo 'Cleaning...' && echo -en 'travis_fold:start:script.3\\r'
   - make check-maintainer-clean && ../../configure $CONFIGOPTS
+  - if test -z "$SWIGLANG"; then cmake -DCMAKE_INSTALL_PREFIX=~/.local ../.. && make install && ~/.local/bin/swig -version; fi
```

Comment:
travis.yml is for the autotools build and has already installed swig at this point. Let's instead just add one separate CMake build into .travis.yml building and installing and uninstalling on Linux.

Reply:
ok